src/components/PeopleFilters.tsx

```diff
@@ -1,19 +1,63 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/getSearchWith';
+import { SearchLink } from './SearchLink';
+
+const centuriesArr = ['16', '17', '18', '19', '20'];
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || null;
+
+  const onQueryChange = (event : React.ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(
+      getSearchWith(searchParams, { query: event.target.value || null }),
```

Comment:
Also you can use 'trimStart()' to avoid pluses appearing in search parameters

Reply:
check @OlhaYevstifieieva comment

Reply:
fix it